valuable/Cargo.toml

```diff
@@ -1,10 +1,25 @@
 [package]
 name = "valuable"
 version = "0.1.0"
-authors = ["Carl Lerche <me@carllerche.com>"]
-edition = "2018"
+authors = ["Tokio Contributors <team@tokio.rs>"]
+edition = "2021"
 license = "MIT"
-description = "Values"
+rust-version = "1.57.0"
```

Comment:
is there a specific feature we use that requires 1.57, or was it just picked for the 2021 edition?

Reply:
I just picked the latest

Reply:
ah, okay, i was just wondering if it would be practical to see what the most conservative MSRV that can build `valuable` is, and go with that. but, it's not a blocker IMO, as tracing doesn't need to bump its MSRV for an off by default feature.